integration/flink/src/main/java/io/openlineage/flink/OpenLineageFlinkJobListener.java

```diff
@@ -44,10 +46,28 @@ public class OpenLineageFlinkJobListener implements JobListener {
 
   public static final String DEFAULT_JOB_NAMESPACE = "flink_jobs";
 
-  /** Default duration between REST API calls for checkpoint stats. */
   static final Duration DEFAULT_TRACKING_INTERVAL = Duration.ofSeconds(10);
 
+  public static final ConfigOption<String> OPEN_LINEAGE_LISTENER_CONFIG_JOB_NAMESPACE =
```

Comment:
Would it be doable and implement that in a similar manner as here: https://github.com/OpenLineage/OpenLineage/blob/1.5.0/integration/spark/app/src/main/java/io/openlineage/spark/agent/ArgumentParser.java#L82C7-L82C7 ?

For Apache Spark, we are translating `spark.conf` entries to `OpenLineageYaml`. The great advantage of this approach is that, whenever a new parameter is added to `openlineage-java`, it is automatically available within Spark integration by setting a proper config. 

My concern is that the list of config params will grow.

Reply:
Thanks for the suggestion.

Then would it make sense to move the job name, namespace parameters inside `OpenLineageYaml` directly?

Then in the builder we can use `FlinkConfigParser.parse(configuration)`  to create an instance of OpenLineageYaml inside the OpenlineageFlinkJobListener?


Please let me know if I misunderstood.

Reply:
My thinking was wrong. Transport config can be already read from flink conf which was introduced with this PR: https://github.com/OpenLineage/OpenLineage/pull/2033

Additionally, `jobName` and `jobNamespace` has to be set directly in flink integration when OL event is built. 

So, my suggestion is wrong. Sorry for that.